home.html

```diff
@@ -206,8 +206,58 @@ <h2>"DJ Night @Felicity2020"</h2>
 								<h3>by Sai Vineesh C</h3>
 								<p>Month: March'20</p>
 							</div>
-						</div>
+						</div> -->
 					</div>
+					<script src="imageDetails.js"></script>
+					<script>
+						// Adding carousel indicators
+						let carouselIndicators = document.getElementsByClassName("carousel-indicators")[0];
```

Comment:
Since there can be a possibility of having multiple carousels, better to use `getElementById` instead. Set the `id` of the element accordingly (e.g. `ceye-carousel-indicators`).